dwave/system/testing.py

```diff
@@ -56,15 +56,25 @@ class MockDWaveSampler(dimod.Sampler, dimod.Structured):
             A list of one number [m] for Pegasus, defaulted as [3].
             A list of two numbers [m,t] for Zephyr, defaulted as [2,4].  
     
-    """
+        parameter_warnings (bool, optional, default=True):
+            The MockSampler is adaptive with respect to ``num_reads``,
+            ``answer_mode`` and ``max_answers`` and ``label`` 
+            parameters. By default ``initial_state`` can also be mocked, if
+            dwave-greedy is installed. All other parameters are ignored and a 
+            warning will be raised by default.
 
+    """
+    # Feature suggestion - add seed as an optional input, to allow reproducibility.
+    
     nodelist = None
     edgelist = None
     properties = None
     parameters = None
-
+    parameter_warnings = True
```

Comment:
This is not needed. We only do this for `nodelist`, `edgelist`, etc to satisfy the abstract base class interface.

Reply:
🌟